ci/common/build.sh

```diff
@@ -46,10 +45,10 @@ build_deps() {
 }
 
 prepare_build() {
-  if [[ -n "${CLANG_SANITIZER}" ]]; then
+  if test -n "${CLANG_SANITIZER}" ; then
     CMAKE_FLAGS="${CMAKE_FLAGS} -DCLANG_${CLANG_SANITIZER}=ON"
   fi
-  if [[ "${BUILD_32BIT}" == ON ]]; then
+  if test "x${BUILD_32BIT}" = xON ; then
```

Comment:
Quoting [this](http://www.pixelbeat.org/programming/shell_script_mistakes.html):
> [ x"$var" = x"find" ] && echo found
The use of x"$var" was required in case var is "" or "-hyphen". Thinking about this for a moment should indicate that the shell can handle both of these cases unambiguously, and if it doesn't it's a bug. This bug was probably fixed about 20 years ago, so stop propagating this nonsense please! Shell doesn't have the cleanest syntax to start with, so polluting it with stuff like this is horrible.